clusterdock/actions/manage.py

```diff
@@ -30,57 +30,84 @@ def main(args):
 
     label_key = defaults.get('DEFAULT_DOCKER_LABEL_KEY')
     if args.manage_action == 'nuke':
-        removed_containers = []
-        containers_networks = []
-        if client.containers.list():
-            logger.info('Stopping and removing '
-                        '{} ...'.format('cluster {} containers'.format(args.cluster_name)
-                                        if args.cluster_name else 'all containers'))
-            for container in client.containers.list(all=True):
+        logger.info('Stopping and removing %s containers ...',
+                    ('all' if args.all else 'clusterdock'))
+        containers_tup = (_get_containers(label_check=False)
```

Comment:
In general, we should avoid putting the data type into the name of variables (https://docs.quantifiedcode.com/python-anti-patterns/readability/putting_type_information_in_a_variable_name.html)